TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -74,61 +80,22 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
         }
 
         /// <summary>
-        /// Constructs protein groups; calculates false discovery rate (FDR) for PSMs and protein groups
+        /// Calculate estimated false-discovery rate (FDR) for peptide spectral matches (PSMs)
         /// </summary>
-        private void ProteinAndFdrAnalysis()
+        private void CalculatePsmFdr()
```

Comment:
This seems like an unnecessary method where instead the FDRAnalysisEngine.Run() could be used instead.